decidim-dev/lib/decidim/dev/assets/participatory_processes.json

```diff
@@ -431,4 +431,4 @@
       }
     ]
   }
-]
\ No newline at end of file
+]
```

Comment:
We should always avoid unnecessary changes in files in all PRs.

Could you please revert this file back to the original version so that we wouldn't change it when merging this PR.

Changing the line ending would be totally fine if you were doing some other changes in this file but since we are not doing any changes to this file, let's keep it at the original version please.